### PR TITLE
Turtle graphics

### PR DESCRIPTION
**Pull request by amirsalaar**
Here is my solution to homework 2

### CODE REVIEW

File: turtle_graphics/index.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        // The number of rows that the turtle has passed:
+        let maxY = Math.max(...transformedPoints.map(element => {
+            return element[1]
+        }).map(Math.abs));
+
+        console.log(transformedPoints)
+
+        for (let i = 0; i <= maxY; i++) { // i refers to row numbers and is checked with y coordinates
+            for (let j = 0; j <= maxX; j++) { // j refers to column numbers and is checked with x coordinates
+                let check = Boolean(false);
```
[the remark this answers is not in the document]
Comment (QuickStyles) [reply turtle_graphics/index.js]: new Boolean(false) has the value of true

```
  const falseObject = new Boolean(false);
  if(falseObject) {
    console.log('true')
  } else {
    console.log('false')
  }
```
will print out "true"

File: turtle_graphics/index.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+    forward(steps) {
+        if (typeof steps !== 'number' || steps < 0) {
+            console.log("Invalid value for steps. \n**Steps are required to be positive integers");
+            process.exit(1);
+        };
+        let tempX = this.x;
+        let tempY = this.y;
+        switch (this.face) {
+            case 'east':
```
[the remark this answers is not in the document]
Comment (amirsalaar) [reply turtle_graphics/index.js]: Very good to know!

[the remark this answers is not in the document]
Comment (amirsalaar) [reply turtle_graphics/index.js]: I just put them outside of the class

File: turtle_graphics/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (amirsalaar) [reply turtle_graphics/index.js]: very good point.
Thank you!

File: turtle_graphics/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (QuickStyles) [reply turtle_graphics/index.js]: Perfect! Usually constants will be top-level at the very top of files just like what you've done

**Pull request merged**